subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformValuesInjectionIntegrationTest.groovy

```diff
@@ -188,6 +203,9 @@ class ArtifactTransformValuesInjectionIntegrationTest extends AbstractDependency
         failure.assertHasCause("Property 'missingInput' does not have a value specified.")
         failure.assertHasCause("Property 'absolutePathSensitivity' is declared to be sensitive to absolute paths. This is not allowed for cacheable transforms.")
         failure.assertHasCause("Property 'noPathSensitivity' is declared without path sensitivity. Properties of cacheable transforms must declare their path sensitivity.")
+        failure.assertHasCause("Property 'nested.outputDirectory' is annotated with unsupported annotation @OutputDirectory.")
+        failure.assertHasCause("Property 'nested.inputFile' is declared without path sensitivity. Properties of cacheable transforms must declare their path sensitivity.")
+        errorOutput.count("> Property") == 2 * 7
```

Comment:
Where does this come from? Can we make it easier to understand by a comment or via some other way?

Reply:
I added a new assertion method which should make it clear.